### ISSUE TITLE
Website: Link to color accessibility PDF doesn't work

### ISSUE THREAD

**Issue by mikewheaton** [OfficeDev/office-ui-fabric-react#1855]
### Bug Report

- __Package version(s)__: N/A
- __Browser and OS versions__: N/A

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Unsure

Requested priority: Normal

Products/sites affected: Fabric Website

#### Describe the issue:

The link to the color accessibility guidance PDF is broken on this page: http://dev.office.com/fabric#/styles/colors

**Comment by Jahnp** [comment]
Closing since this has been resolved.
